verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -1124,18 +1169,32 @@ components:
       properties:
         country:
           type: string
+          description: English full name of a country
         administrativeAreaLevel1:
           type: string
+          description: First administrative subdivision of a country
         administrativeAreaLevel2:
           type: string
+          description: Second administrative subdivision of a country
         administrativeAreaLevel3:
           type: string
+          description: Third administrative subdivision of a country
         place:
           type: string
+          description: Name of the place this location refers to
+          example: Boston Children's Hospital
         name:
           type: string
+          description: >
+            Name that is enough describe by itself the location most
```

Comment:
I'm not parsing this sentence, could you reword?

Reply:
done